db/migrate/20240910012046_add_username_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddUsernameToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :username, :string
+    add_index :users, :username, uniqe: true
```

Comment:
typoがあるのでたぶん `db:migrate` を実行するとエラーが発生してしまいます。

```suggestion
    add_index :users, :username, unique: true
```